.github/workflows/scripts/download-nvd.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# First argument: Year
+YEAR=$1
+
+# Paths
+META_PATH="nvddata/nvdcve-1.1-${YEAR}.meta"
+JSON_GZ_PATH="nvddata/nvdcve-1.1-${YEAR}.json.gz"
+JSON_PATH="nvddata/nvdcve-1.1-${YEAR}.json"
+
+# Create the nvddata directory
+mkdir -p nvddata
+
+# Fetch the meta file for the given year
+curl --fail --silent --show-error --max-time 60 --retry 3 -o "$META_PATH" https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-"$YEAR".meta || (echo "Error fetching meta file for year $YEAR" && exit 1)
```

Comment:
Just few nits: 
- You could refactor the URL into a var, `https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-` is same for both requests
- You could turn the requests into a function, since both curl commands use the same options